base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {ebsx}{el} {elsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{l}  {lsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sl} {slsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{sb} {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{b}  {bsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{eb} {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{ub} {ubsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebsx}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebsx}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebsx}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebsx}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebx}{ul}  {ulx}  {}
+\DeclareFontSeriesChangeRule {ebx}{el}  {elx}  {}
+\DeclareFontSeriesChangeRule {ebx}{l}   {lx}   {}
+\DeclareFontSeriesChangeRule {ebx}{sl}  {slx}  {}
+\DeclareFontSeriesChangeRule {ebx}{sb}  {sbx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{b}   {bx}   {}
+\DeclareFontSeriesChangeRule {ebx}{eb}  {ebx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{ub}  {ubx}  {bx}
+\DeclareFontSeriesChangeRule {ebx}{uc}  {ebuc} {}
+\DeclareFontSeriesChangeRule {ebx}{ec}  {ebec} {}
+\DeclareFontSeriesChangeRule {ebx}{c}   {ebc}  {}
+\DeclareFontSeriesChangeRule {ebx}{sc}  {ebsc} {}
+\DeclareFontSeriesChangeRule {ebx}{sx}  {ebsx} {}
+\DeclareFontSeriesChangeRule {ebx}{x}   {ebx}  {}
+\DeclareFontSeriesChangeRule {ebx}{ex}  {ebex} {}
+\DeclareFontSeriesChangeRule {ebx}{ux}  {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebex}{ul} {ulex} {}
+\DeclareFontSeriesChangeRule {ebex}{el} {elex} {}
+\DeclareFontSeriesChangeRule {ebex}{l}  {lex}  {}
+\DeclareFontSeriesChangeRule {ebex}{sl} {slex} {}
+\DeclareFontSeriesChangeRule {ebex}{sb} {sbex} {bex}
+\DeclareFontSeriesChangeRule {ebex}{b}  {bex}  {}
+\DeclareFontSeriesChangeRule {ebex}{eb} {ebex} {}
+\DeclareFontSeriesChangeRule {ebex}{ub} {ubex} {bex}
+\DeclareFontSeriesChangeRule {ebex}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebex}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebex}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebex}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebex}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebex}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebex}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebex}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebux}{ul} {ulux} {}
+\DeclareFontSeriesChangeRule {ebux}{el} {elux} {}
+\DeclareFontSeriesChangeRule {ebux}{l}  {lux}  {}
+\DeclareFontSeriesChangeRule {ebux}{sl} {slux} {}
+\DeclareFontSeriesChangeRule {ebux}{sb} {sbux} {bux}
+\DeclareFontSeriesChangeRule {ebux}{b}  {bux}  {}
+\DeclareFontSeriesChangeRule {ebux}{eb} {ebux} {bux}
+\DeclareFontSeriesChangeRule {ebux}{ub} {ubux} {bux}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebux}{ub} {ubux} {ebux}
```

I think it is better to use `eb` instead of `b` in the fourth argument: `eb` is a better approximation for `ub` and since the fourth argument is then identical to the first, we can assume that this font face actually exists.